.circleci/config.yml

```diff
@@ -552,7 +556,7 @@ workflows:
     jobs:
       - auth_smoke_tests:
           name: "auth_smoke_tests_dev"
```

Comment:
Can you please update `dev` to `qa`? Look like I missed both of these in my PR (also please update line 532).